src/Storages/LiveView/LiveViewBlockOutputStream.h

```diff
@@ -2,6 +2,7 @@
 
 #include <DataTypes/DataTypesNumber.h>
 #include <DataStreams/IBlockOutputStream.h>
+#include <DataStreams/PushingToViewsBlockOutputStream.h>
```

Comment:
Don't see where it is needed.

Reply:
Yes, I think you are right and I can remove it.